ee/clickhouse/models/person.py

```diff
@@ -113,6 +113,36 @@ def delete_person(
     sync_execute(DELETE_PERSON_BY_ID, data)
 
 
+# counts duplicate distinct IDs in clickhouse for a team broken down by day
```

Comment:
Doesn't this work on Postgres too?

Reply:
no such thing as duplicates in Postgres, so this is a CH-only metric.

Reply:
nit this comment is out of date now